script/build-ubuntu.sh

```diff
@@ -128,12 +146,13 @@ check_static_linking "$DESTINATION"
 
 set -eu -o pipefail
 
+if [ "$TARGET_ARCH" == "x64" ]; then
```

Comment:
Why only x64?

Reply:
the native github runner is x64. as such, it can only run x64 code natively. so the tests are only performed on x64 builds and skipped on architectures that are not native to the runner